### ISSUE TITLE
Refactor translations to use nesting

### ISSUE THREAD

**Issue by with-heart** [Developer-DAO/developer-dao#91]
I don't think we have a pressing need for this, but it'd be nice to establish [translation nesting](https://www.i18next.com/translation-function/nesting) as a pattern sooner rather than later. 

The example I had in mind is refactoring the many uses of "Developer DAO" in the various translations to refer to the same `title` string rather than duplicating it all over the place:

```diff
 {
   "title": "Developer DAO",
-  "searchId": "Search developer DAO id",
+  "searchId": "Search $t(title) id",
   "loading": "Loading...",
   "error": "Error",
-  "madeBy": "Made by the developer DAO community: ",
+  "madeBy": "Made by the $t(title) community: ",
   "owner": "Owner",
   "unclaimed": "Unclaimed",
-  "enterDeveloperId": "Please enter a developer DAO id",
+  "enterDeveloperId": "Please enter a $t(title) id",
   "hosting": "Hosting by:",
   "copyLinkToNFT": "Copy link to NFT",
   "linkCopied": "Link copied to clipboard"
}
```

This pattern would push us in the direction of defining a glossary of foundational terms that we then re-use in more complex strings.

**Comment by codingwithmanny** [comment]
I like "Nesting" not only because it accomplishes what it does but also because it sounds a whole lot shorter and nicer than "String Interpolation" 😂

**Comment by TrevorFrench** [comment]
Could I work on this one?

**Comment by codingwithmanny** [comment]
@TrevorFrench all yours

**Comment by codingwithmanny** [comment]
Closes this because it was merged in #104 thanks again @TrevorFrench
